pkg/udf/reduce_udf.go

```diff
@@ -52,9 +54,18 @@ func (u *ReduceUDFProcessor) Start(ctx context.Context) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	var windower window.Windower
-	if x := u.VertexInstance.Vertex.Spec.UDF.GroupBy.Window.Fixed; x != nil {
-		windower = fixed.NewFixed(x.Length.Duration)
+	f := u.VertexInstance.Vertex.Spec.UDF.GroupBy.Window.Fixed
+	s := u.VertexInstance.Vertex.Spec.UDF.GroupBy.Window.Sliding
+
+	if f != nil && s != nil {
```

Comment:
Can be removed since it is already validated by the controller.

Reply:
removed.